ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -156,8 +169,31 @@ function DeleteDialogContentUI(props: DeleteDialogContentUIProps) {
               InputProps={{
                 className: classes.textField
               }}
+              inputProps={{ maxLength: submissionCommentCount.maxLength }}
```

Comment:
submissionCommentCount is not a good name for the var that contains maxLength. also, consider just having a var for that specifically; what's the value of having it in an object with other stuff?